api/src/main/java/net/wesjd/anvilgui/AnvilGUI.java

```diff
@@ -229,10 +245,47 @@ public void onInventoryClick(InventoryClickEvent event) {
             final int rawSlot = event.getRawSlot();
             if (rawSlot < 3 || event.getAction().equals(InventoryAction.MOVE_TO_OTHER_INVENTORY)) {
                 event.setCancelled(!interactableSlots.contains(rawSlot));
-                final List<ResponseAction> actions =
+                if (clickHandlerRunning) {
```

Comment:
Based on our previous discussion: should we make this opt-out from the `Builder` side if the developer is fine with multiple running at the same time?